src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './component/calculator';
 
 function App() {
   return (
```

Comment:
- [ ] App.js should be a class-based component as per requirement. Kindly fix this

Reply:
App.js is now a class-based component